core/bin/zksync_server/src/node_builder.rs

```diff
@@ -364,6 +364,7 @@ impl MainNodeBuilder {
             subscriptions_limit: Some(rpc_config.subscriptions_limit()),
             batch_request_size_limit: Some(rpc_config.max_batch_request_size()),
             response_body_size_limit: Some(rpc_config.max_response_body_size()),
+            with_extended_tracing: rpc_config.extended_api_tracing,
```

Comment:
Maybe this was skipped intentionally, but I don't see a particular reason why the WS server should have extended tracing and the HTTP one shouldn't.

Reply:
Either I missed it, or it was configured this way in the old API server configuration code (there were other desynchronized configs in there too). Either way, it was not intentional.